frontend/src/components/mine/Permit/MinePermitInfo.js

```diff
@@ -24,40 +24,66 @@ const columns = [
     title: "Permit No.",
     dataIndex: "permitNo",
     key: "permitNo",
+    render: (text) => <div title="Permit No.">{text}</div>,
```

Comment:
This allows CSS to access the title attribute and add this as a label for responsive table views.